types/ioredis/index.d.ts

```diff
@@ -229,7 +229,7 @@ declare namespace IORedis {
         smembers(key: KeyType, callback: (err: Error, res: any) => void): void;
         smembers(key: KeyType): Promise<any>;
 
-        zadd(key: KeyType, ...args: string[]): Promise<number | string>;
+        zadd(key: KeyType, ...args: string | number[]): Promise<number | string>;
```

Comment:
This needs to be `Array<string | number>` to have both `string` and `number` as possible values in the array